### PR TITLE
NH-15: add link to devpost

### PR DESCRIPTION
**Pull request by lukecheseldine**
## Overview

- Resolves NH-15
- added placeholder for link to devpost

## Unit Tests Created

- none

## Steps to QA

- you might have to `yarn run scss` and `python manage.py collectstatic`, not sure how it works lol
- go to the landing page and look at the link :)
Screen Shot 2022-09-01 at 12 32 00 AM

### CODE REVIEW

**Comment by lukecheseldine** [comment]
i pushed at midnight rerun tests pls. also is there an actual link to the 2021 devpost?

**Review by Mustaballer — COMMENTED** [review]
The devpost link for 2021 is this: https://newhacks-2021.devpost.com/

File: hackathon_site/event/jinja2/event/landing.html [hunk 1]
```diff
@@ -325,7 +325,11 @@ <h3 class="center white-text">Sponsors</h3>
             </p>
         </div>
     </div>
+    <div class="container section scrollspy" id="past-projects">
```
Comment (YuYing-Liang) [inline hackathon_site/event/jinja2/event/landing.html:328]: Could you increase the space between this past projects text and the carousel

**Review by YuYing-Liang — COMMENTED** [review]
Looks good Luke, just a small suggestion

File: hackathon_site/event/jinja2/event/landing.html (hunk 1 above)
Comment (lukecheseldine) [reply hackathon_site/event/jinja2/event/landing.html:328]: added 40px bottom padding

**Review by YuYing-Liang — APPROVED** [review]

**Pull request merged**